frontend/.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+    "printWidth": 120,
+    "singleQuote": true,
+    "useTabs": false,
+    "tabWidth": 2,
+    "semi": true,
+    "bracketSpacing": true,
```

Comment:
Just curious what exactly does `"bracketSpacing": true,` do?

Reply:
@briandavidson - I wasn't really sure either. I pulled it in as a suggestion from the prettier tutorial I used. Here's what I found though:

![image](https://user-images.githubusercontent.com/23467847/102530655-e7849600-4066-11eb-8868-bf1ecf69fddb.png)

Reply:
What are your thoughts? BracketSpacing true or false?

Reply:
> What are your thoughts? BracketSpacing true or false?

I personally prefer `false`. @mtlaney ?

Reply:
@briandavidson - I like false too. Imma make the switch. We can easily switch it back.